erpnext/projects/doctype/timesheet/timesheet.py

```diff
@@ -297,3 +299,32 @@ def get_employee_list(doctype, txt, searchfield, start, page_len, filters):
 		from `tabSalary Structure` where salary_slip_based_on_timesheet=1
 		and employee like %(txt)s limit %(start)s, %(page_len)s""", 
 		{'txt': "%%%s%%"%(txt), 'start': start, 'page_len': page_len})
+		
+@frappe.whitelist()
+def get_events(start, end, filters=None):
+	"""Returns events for Gantt / Calendar view rendering.
+	:param start: Start date-time.
+	:param end: End date-time.
+	:param filters: Filters (JSON).
+	"""
+	from frappe.desk.calendar import get_event_conditions
+	filters = json.loads(filters)
```

Comment:
Not sure if there is a better way to pull Employee our of the filters, because you don't want the standard get_event_conditions to process it (because you need to look in the parent view instead of the child).